Scripts/Multis/BaseHouse.cs

```diff
@@ -422,6 +422,27 @@ public virtual int GetAosMaxLockdowns()
             return (int)(hpe.Lockdowns * BonusStorageScalar);
         }
 
+        private Type[] _Table = new Type[]
```

Comment:
Rename _Table to something a bit more descriptive, ie _NoItemCountTable or something.